tests/test_weaver_snapshot_serving.py

```
@@ -122,6 +119,101 @@ def probe(_port: int) -> None:
+        """Report still running."""
+        return None
+
+    def terminate(self) -> None:
+        """Record the request to stop."""
+        self.stopped = True
+
+    def kill(self) -> None:
+        """Record the insistence, the same way."""
+        self.stopped = True
+
+    def wait(self, timeout: int | None = None) -> int:
```

Ruff (0.16.2)

[warning] Unused method argument: `timeout`

(ARG002)